contributing/contribution_guide.rst

```diff
@@ -0,0 +1,128 @@
+How to contribute ?
+===================
+
+As Akeneo PIM is an open-source project all contribution is very welcome.
```

Comment:
As i think.
~ all contributions are very welcome 
or
~ any contribution is very welcome